scripts/prepare_sector_network.py

```diff
@@ -4373,6 +4378,12 @@ def add_enhanced_geothermal(n, egs_potentials, egs_overlap, costs):
     )
     pop_weighted_energy_totals.update(pop_weighted_heat_totals)
 
+    heating_efficiencies = (
+        (pd.read_csv(snakemake.input.heating_efficiencies, index_col=[0, 1]))
+        .swaplevel()
+        .loc[int(snakemake.config["energy"]["energy_totals_year"])]
+    )
```

Comment:
similar as above; no `snakemake.config` due to scenario management.

```suggestion
    fn = snakemake.input.heating_efficiencies
    year = int(snakemake.params["energy_totals_year"])
    heating_efficiencies = pd.read_csv(fn, index_col=[0, 1]).swaplevel().loc[year]
```

And add to `perpare_sector_network` rule definition:

```
params:
    energy_totals_year=config_provider("energy", "energy_totals_year")
```